test/integration/Cardano/Wallet/Network/HttpBridgeSpec.hs

```diff
@@ -89,6 +90,7 @@ spec = do
     newNetworkLayer =
         HttpBridge.newNetworkLayer "testnet" port
     startBridge = do
+        threadDelay 1000000
```

Comment:
@KtorZ I added this to avoid `Address already in use`-problems. Should pass now, but not exactly sure what the problem was.

A failing log: https://travis-ci.org/input-output-hk/cardano-wallet/jobs/510990561

Reply:
10 seconds is a rather long delay, and I wouldn't expect this to fail even without any delay between two successive runs of the spec.

Reply:
:thinking: ... having a look

Reply:
I thought that was 1s 😅

Reply:
So yeah, it seems that we `cancel` from the `async` seems to wait for the delivery of the exception, but the socket isn't freed immediately by the OS. I've added a delay of 500ms which should be plenty of time already to free up the socket. I've re-run that 10x in a row without getting any issues :thinking: